scripts/vet.sh

```diff
@@ -1,13 +1,18 @@
 #!/bin/bash
 
-set -ex  # Exit on error; debugging enabled.
-set -o pipefail  # Fail a pipe if any sub-command fails.
+set -ex         # Exit on error; debugging enabled.
+set -o pipefail # Fail a pipe if any sub-command fails.
 
 source "$(dirname $0)/vet-common.sh"
 
 # Check to make sure it's safe to modify the user's git repo.
 git status --porcelain | fail_on_output
 
+# Suppresses the return code of grep since grep would fail vet when there are no matches (for eg, empty file).
+noret_grep() {
+  grep "$@" || :
```

Comment:
Like we did with diff, should this only suppress if the exit code is 1 ("no lines selected")?  i.e. `|| [[ $? == 1 ]]` instead of `|| :`?

Reply:
Nice catch. Thanks! Fixed